cpp/models/lct_secir/parameters_io.h

```diff
@@ -0,0 +1,67 @@
+/* 
+* Copyright (C) 2020-2024 MEmilio
+*
+* Authors: Lena Ploetzke
+*
+* Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#ifndef LCTSECIR_PARAMETERS_IO_H
+#define LCTSECIR_PARAMETERS_IO_H
+
+#include "memilio/config.h"
+
+#ifdef MEMILIO_HAS_JSONCPP
+
+#include "lct_secir/parameters.h"
+#include "lct_secir/infection_state.h"
+#include "memilio/math/eigen.h"
+
+#include <string>
+
+namespace mio
+{
+namespace lsecir
+{
+
+/**
+* @brief Computes an initialization vector for a LCT model with data from RKI.
+*   
+* Computes an initial value vector for an LCT model with the defined infectionState and the parameters. 
+* For the computation expected sojourntime in the subcompartments are used. To calculate the initial values, 
```

Comment:
```suggestion
* For the computation expected sojourn times in the subcompartments are used. To calculate the initial values, 
```
Do you always use the term sojourn time in the LCT model or do you use mean stay time as well?

Reply:
Always used sojourn time but maybe stay time is also ok and easier to understand. Changed it.